examples/src/main/java/io/grpc/examples/experimental/CompressionServerInterceptor.java

```diff
@@ -0,0 +1,18 @@
+package io.grpc.examples.experimental;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCall.Listener;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+
+public class CompressionServerInterceptor implements ServerInterceptor {
```

Comment:
What do you think about inlining this class into CompressingHelloWorldServerAllMethods? It's so small/simple, I wonder if it would be better to just put it all in the same file.

Reply:
Good point. Use it as an anonymous class now. An inner class is also good.